wrapper/src/main/java/software/amazon/jdbc/plugin/efm/HostMonitoringConnectionPlugin.java

```diff
@@ -298,7 +272,8 @@ private Connection connectInternal(String driverProtocol, HostSpec hostSpec,
     final Connection conn = connectFunc.call();
 
     if (conn != null) {
-      generateHostAliases(driverProtocol, conn, hostSpec);
+      hostSpec.resetAliases();
```

Comment:
Why to reset? Some other plugin might already filled aliases for this host spec.